canopen/objectdictionary/eds.py

```diff
@@ -36,6 +36,17 @@ def import_eds(source, node_id):
         od.node_id = int(eds.get("DeviceComissioning", "NodeID"))
 
     for section in eds.sections():
+        # Match dummy definitions
+        match = re.match(r"^[D|d]ummy[U|u]sage$", section)
```

Comment:
You don’t need the | here. All characters inside [] will be matched.

Reply:
Right, will remove the "pipes"